[2021-06-06 17:26:08,656] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: create_rox_tables.create_person_table 2021-06-06T17:26:03.634299+00:00 [queued]>
[2021-06-06 17:26:08,672] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: create_rox_tables.create_person_table 2021-06-06T17:26:03.634299+00:00 [queued]>
[2021-06-06 17:26:08,672] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-06 17:26:08,672] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-06 17:26:08,674] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-06 17:26:08,682] {taskinstance.py:1087} INFO - Executing <Task(PostgresOperator): create_person_table> on 2021-06-06T17:26:03.634299+00:00
[2021-06-06 17:26:08,689] {standard_task_runner.py:52} INFO - Started process 9047 to run task
[2021-06-06 17:26:08,692] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'create_rox_tables', 'create_person_table', '2021-06-06T17:26:03.634299+00:00', '--job-id', '19', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/create_rox_tables.py', '--cfg-path', '/tmp/tmp1pmocavf', '--error-file', '/tmp/tmp7wndklkl']
[2021-06-06 17:26:08,693] {standard_task_runner.py:77} INFO - Job 19: Subtask create_person_table
[2021-06-06 17:26:08,725] {logging_mixin.py:104} INFO - Running <TaskInstance: create_rox_tables.create_person_table 2021-06-06T17:26:03.634299+00:00 [running]> on host 21bd802337aa
[2021-06-06 17:26:08,759] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_rox_tables
AIRFLOW_CTX_TASK_ID=create_person_table
AIRFLOW_CTX_EXECUTION_DATE=2021-06-06T17:26:03.634299+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-06T17:26:03.634299+00:00
[2021-06-06 17:26:08,759] {postgres.py:70} INFO - Executing: 
    CREATE TABLE IF NOT EXISTS Person
    (
        BusinessEntityID INTEGER NOT NULL,
        PersonType VARCHAR(10),
        NameStyle INTEGER,
        Title VARCHAR(10),
        FirstName VARCHAR(50),
        MiddleName VARCHAR(50),
        LastName VARCHAR(50),
        Suffix VARCHAR(10),
        EmailPromotion INTEGER,
        AdditionalContactInfo VARCHAR(2000),
        Demographics VARCHAR(1000),
        rowguid VARCHAR(36),
        ModifiedDate TIMESTAMP,
        PRIMARY KEY(BusinessEntityID)
    )
[2021-06-06 17:26:08,765] {base.py:78} INFO - Using connection to: id: redshift. Host: doxcluster.craeubgpe2r5.us-east-1.redshift.amazonaws.com, Port: 5439, Schema: dox, Login: doxuser, Password: ***, extra: {}
[2021-06-06 17:26:11,486] {dbapi.py:204} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS Person
    (
        BusinessEntityID INTEGER NOT NULL,
        PersonType VARCHAR(10),
        NameStyle INTEGER,
        Title VARCHAR(10),
        FirstName VARCHAR(50),
        MiddleName VARCHAR(50),
        LastName VARCHAR(50),
        Suffix VARCHAR(10),
        EmailPromotion INTEGER,
        AdditionalContactInfo VARCHAR(2000),
        Demographics VARCHAR(1000),
        rowguid VARCHAR(36),
        ModifiedDate TIMESTAMP,
        PRIMARY KEY(BusinessEntityID)
    ), parameters: None
[2021-06-06 17:26:12,419] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=create_rox_tables, task_id=create_person_table, execution_date=20210606T172603, start_date=20210606T172608, end_date=20210606T172612
[2021-06-06 17:26:12,436] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-06 17:26:12,441] {local_task_job.py:151} INFO - Task exited with return code 0
